src/libraries/System.Security.Cryptography.Encoding/ref/System.Security.Cryptography.Encoding.cs

```diff
@@ -12,10 +12,11 @@ public partial class AsnEncodedData
         protected AsnEncodedData() { }
         public AsnEncodedData(byte[] rawData) { }
         public AsnEncodedData(System.Security.Cryptography.AsnEncodedData asnEncodedData) { }
-        public AsnEncodedData(System.Security.Cryptography.Oid oid, byte[] rawData) { }
+        public AsnEncodedData(System.Security.Cryptography.Oid? oid, byte[] rawData) { }
         public AsnEncodedData(string oid, byte[] rawData) { }
-        public System.Security.Cryptography.Oid Oid { get { throw null; } set { } }
-        public byte[] RawData { get { throw null; } set { } }
+        public System.Security.Cryptography.Oid? Oid { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.DisallowNullAttribute]
+        public byte[]? RawData { get { throw null; } set { } }
```

Comment:
I don't think there's a legitimate case for the get to return null; I'd just call this non-nullable.

Reply:
Good catch it was actually not init-ed only in protected constructor which not used